src/components/Button.js

```diff
@@ -21,4 +20,6 @@ export const Button = (props) => {
 
 Button.propTypes = {
   name: PropTypes.string.isRequired,
+  color: PropTypes.string.isRequired,
```

Comment:
- [ ] I got a console error here:
![Screenshot from 2021-01-14 22-15-38](https://user-images.githubusercontent.com/4246637/104644192-1f0e4e80-56b6-11eb-9d46-3ea5a22da928.png)

Make sure to pass the color prop or declare a default prop [declaring default props](![Screenshot from 2021-01-14 22-15-38](https://user-images.githubusercontent.com/4246637/104644192-1f0e4e80-56b6-11eb-9d46-3ea5a22da928.png))